packages/docusaurus-1.x/lib/core/DocsSidebar.js

```diff
@@ -10,6 +10,8 @@ const fs = require('fs');
 const Container = require('./Container.js');
 const SideNav = require('./nav/SideNav.js');
 const Metadata = require('../core/metadata.js');
+
+const siteConfig = require(`${process.cwd()}/siteConfig.js`);
```

Comment:
https://github.com/facebook/Docusaurus/blob/8f6b5b8e20f9efa3998378fcc2160d836c5257e2/packages/docusaurus-1.x/lib/core/DocsLayout.js

Can we get the config from passed props ? DocsLayout has config as its props, and we can pass it to DocsSidebar

Reply:
Hmm ok sure, edited!